doc/api/http.md

```diff
@@ -772,6 +772,8 @@ server — in that case it is suggested to use the
 `['Transfer-Encoding', 'chunked']` header line when
 creating the request.
 
+Note that this module automatically encode data into chunked encoding.
```

Comment:
```suggestion
This module automatically encodes data in chunked encoding.
```

(That's a grammar fix and a style fix, but the content is still problematic, I think.)